.github/workflows/publish-dev.yml

```diff
@@ -4,8 +4,8 @@ on:
     - cron: '0 */12 * * *'
   workflow_dispatch:
     inputs:
-      pull:
-        description: 'The pull number to check out'
+      ref:
+        description: 'The ref to check out'
```

Comment:
Might be useful to include examples here

Reply:
will do